web/src/Pages/Analysis.js

```diff
@@ -451,38 +482,54 @@ class Analysis extends React.Component {
             alignItems="stretch"
 
           >
-            <Grid item lg={3} xs={6}>
-              <Paper elevation={3} className={classes.paper}>
-                <div className={classes.paper_div}>
-                  <h6 align="left" className={classes.paper_title}>Total Tracks</h6>
-                  <h1 align="left" className={classes.paper_title}>{this.state.playlist.tracks.total}</h1>
-                </div>
-              </Paper>
-            </Grid>
-            <Grid item lg={3} xs={6}>
+            <Grid item lg={4} xs={6}>
               <Paper elevation={3} className={classes.paper}>
                 <div className={classes.paper_div}>
                  <h6 align="left" className={classes.paper_title}>Followers</h6>
                  <h1 align="left" className={classes.paper_title}>{this.state.playlist.followers.total}</h1>
                 </div>
               </Paper>
             </Grid>
-            <Grid item lg={3} xs={6}>
+            <Grid item lg={4} xs={6}>
               <Paper elevation={3} className={classes.paper}>
                 <div className={classes.paper_div}>
                 <h6 align="left" className={classes.paper_title}>Playlist Owner</h6>
                 <PlaylistOwner align="left" className={classes.paper_title}>{this.state.playlist.owner.id}</PlaylistOwner>
                 </div>
               </Paper>
             </Grid>
-            <Grid item lg={3} xs={6} align="stretch">
+            <Grid item lg={4} xs={6} align="stretch">
               <Paper elevation={3} className={classes.paper}>
                 <div className={classes.paper_div}>
                   <h6 align="left" className={classes.paper_title}>Last Updated</h6>
                   <h2 align="left" className={classes.paper_title}>{this.state.last_update.slice(0,nthIndex(this.state.last_update,' ',4))}</h2>
                 </div>
               </Paper>
             </Grid>
+            <Grid item lg={4} xs={6}>
+              <Paper elevation={3} className={classes.paper}>
+                <div className={classes.paper_div}>
+                  <h6 align="left" className={classes.paper_title}>Total Tracks</h6>
+                  <h1 align="left" className={classes.paper_title}>{this.state.playlist.tracks.total}</h1>
+                </div>
+              </Paper>
+            </Grid>
+            <Grid item lg={4} xs={6}>
+              <Paper elevation={3} className={classes.paper}>
+                <div className={classes.paper_div}>
+                  <h6 align="left" className={classes.paper_title}>Total Playlistlength</h6>
```

Comment:
Just a small typo - could you change **Total Playlistlength** to **Total Playlist Length**?